tiles/sprites/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:16.04
+
+RUN apt-get -yqq update && apt-get -yqq install python3
+
+COPY . /var/app
+WORKDIR /var/app
+
+EXPOSE 7800
+
+CMD ["python3", "simple-cors-http-server.py", "7800"]
```

Comment:
This could probably be replaced by a node-based server in the other container, I don't know if webpack-dev-server supports CORS, but this one does for sure: https://www.npmjs.com/package/http-server